cmd/util/cmd/read-hotstuff/cmd/root.go

```diff
@@ -30,6 +32,11 @@ func init() {
 	rootCmd.PersistentFlags().StringVarP(&flagDatadir, "datadir", "d", "/var/flow/data/protocol", "directory to the badger dababase")
 	_ = rootCmd.MarkPersistentFlagRequired("datadir")
 
+	rootCmd.PersistentFlags().StringVar(&flagPebbleDir, "pebble-dir", "/var/flow/data/protocol-pebble", "directory to the pebble dababase")
```

Comment:
This allows the data to be stored in pebble and queried from pebble in the future.